charts/app-config/values-production.yaml

```
@@ -2846,6 +2846,8 @@ govukApplications:
             secretKeyRef:
               name: support-zendesk
               key: ticket_email
+        - name: GOOGLE_TAG_MANAGER_ID
+          value: *publishing-design-system-gtm-id
```

Support app is not fully using Design System yet. Only the first page was migrated.
```suggestion
          value: **publishing-bootstrap-gtm-id
```

---

Ah okay this did confused me - thanks @AgaDufrat !